client/cli/src/runner.rs

```diff
@@ -176,14 +183,20 @@ impl<C: SubstrateCli> Runner<C> {
 
 	/// A helper function that runs a future with tokio and stops if the process receives the signal
 	/// `SIGTERM` or `SIGINT`.
-	pub fn run_subcommand<B, BC, BB>(self, subcommand: &Subcommand, builder: B) -> Result<()>
+	pub fn run_subcommand<RA, B, BC, BB>(self, subcommand: &Subcommand, builder: B) -> Result<()>
```

Comment:
If I am not running a FRAME-based Runtime, the other commands must still be available. I am afraid this must be refactored to allow for that...